examples/full-example/package.json

```diff
@@ -60,10 +59,12 @@
     "mendel-pipeline": "*",
     "mendel-transform-babel": "*",
     "mendel-transform-less": "*",
+    "mocha": "^2.5.0",
     "mocha-istanbul-reporter": "^0.1.0",
     "nodemon": "^1.9.2",
     "react-addons-test-utils": "^15.4.2",
-    "rtlcss": "^2.1.2"
+    "rtlcss": "^2.1.2",
+    "watchify": "^3.0.0"
```

Comment:
It was a dependency for `mendel-development-middleware@2.1.1`. I have removed it now, as it is not required for `mendel-development-middleware@3.0.0-beta.2`.